nflows/distributions/base.py

```diff
@@ -47,13 +47,16 @@ def sample(self, num_samples, context=None, batch_size=None):
 
         Args:
             num_samples: int, number of samples to generate.
-            context: Tensor or None, conditioning variables. If None, the context is ignored.
+            context: Tensor or None, conditioning variables. If None, the context is ignored. 
+                     Should have shape [context_size, context_features], which will generate
```

Comment:
Are we sure this isn't `[context_size, ...]`, i.e. context can't have an arbitrary shape?